src/models/thelio-mira-r1.0/internal-overview.md

```diff
@@ -0,0 +1,26 @@
+# Thelio Mira (Internal Overview)
+
+## Side overview:
+
+![Internal overview (side)](./img/internal-labeled.png)
+
+- CPU cooler is highlighted in blue
+- RAM slots are highlighted in red
+- PCIe slots are highlighted in green
```

Comment:
Is there a reason we're not highlighting/listing the x1 slots? GPUs would typically only use the x16 slots, but the two x1 slots should be usable depending on the GPU configuration, right?

Reply:
I just didn't think it was important, but I can add that. I still have my original SVG, so I'll make the modifications and push a new image.